build_tools/azure/install.sh

```diff
@@ -120,7 +119,15 @@ except ImportError:
 "
 python -m pip list
 
-# Use setup.py instead of `pip install -e .` to be able to pass the -j flag
-# to speed-up the building multicore CI machines.
-python setup.py build_ext --inplace -j 3
-python setup.py develop
+if [[ "$DISTRIB" == "conda-pip-latest" ]]; then
+    # Check that pip can automatically install the build dependencies from
+    # pyproject.toml using an isolated build environment:
+    pip install --verbose --editable .
+else
+    # Use the pre-installed build dependencies and build directly in the
+    # current environment.
+    # Use setup.py instead of `pip install -e .` to be able to pass the -j flag
+    # to speed-up the building multicore CI machines.
+    python setup.py build_ext --inplace -j 3
+    python setup.py develop
+fi
```

Comment:
Nit

Reply:
@NicolasHug you did not finish your comment, or am I missing something ?

Reply:
there was a missing line return but it's been fixed (it showed on the main diff, maybe not in the PR comments page)

Reply:
oh I see. I must have fixed that in a recent commit indeed :)